### DISCUSSION TITLE
Problem: ansible-semaphore-9.5.0 2.10 - bad boolean environment value '0

### DISCUSSION THREAD

**Discussion by monique77** [semaphoreui/semaphore#2116] [Q&A]
### Issue

Hello, Guys

I would like your help, if possible. So, I'm trying to configure a repository in my github through of ansible semaphore, but it has a error bellow;

I configured the key private, and I putted the public key in my github. In my environment has {} but I tried to put { "environment": "localhost"
}
But I have the same error. In the documentation I didn't see this specific error.
I will appreciate your help if possible you help me. This installation was by helm chart in my cluster kubernetes
Tks so much

Versions
kubernetes v1.30.1
ansible-semaphore-9.5.0 2.10
Helm v3.15.1

1:55:09 PM
Cloning into 'repository_4_7'...
1:55:09 PM
fatal: bad boolean environment value '0
1:55:09 PM
' for 'GIT_TERMINAL_PROMPT'
1:55:09 PM
Failed updating repository: exit status 128
1:55:09 PM
Running playbook failed: exit status 128

### Impact

Ansible (task execution)

### Installation method

Package

### Database

MySQL

### Browser

Firefox

### Semaphore Version

ansible-semaphore-9.5.0 2.10

### Ansible Version

```bash
ansible-semaphore-9.5.0 2.10
```

### Logs & errors

![ansibleSemaphore](https://github.com/semaphoreui/semaphore/assets/7231409/2ed0ff6d-4807-4bb7-954a-9e54e95f7157)

### Manual installation - system information

DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=22.04
DISTRIB_CODENAME=jammy
DISTRIB_DESCRIPTION="Ubuntu 22.04.4 LTS"

### Configuration

![repository](https://github.com/semaphoreui/semaphore/assets/7231409/c6e0c209-75ea-453c-9e86-580f9375d2df)

**Comment by FruitPassion** [comment]
This seem to happen when a `login_password` type of key dont have a proper login and only a password although it's optionnal
